conda/recipes/libcudf/meta.yaml

```diff
@@ -29,16 +29,19 @@ requirements:
   host:
     - librmm {{ minor_version }}.*
     - cudatoolkit {{ cuda_version }}.*
-    - arrow-cpp 0.15.0.*
+    - arrow-cpp 0.17.1.*
+    - arrow-cpp-proc * cuda
     - double-conversion
     - rapidjson
     - flatbuffers
     - boost-cpp
     - dlpack
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
-    - arrow-cpp 0.15.0.*
+    - arrow-cpp 0.17.1.*
```

Comment:
`arrow-cpp` [uses `run_exports`]( https://github.com/conda-forge/arrow-cpp-feedstock/blob/9907e672722ab8516e5714ea0f07f4d5807e9177/recipe/meta.yaml#L46-L47 ). So it will already be added as a dependency and pinned based on the version used during `host`. As a result we can just drop this.

```suggestion
```